### PR TITLE
Add git_guide.md and update it

### PR DESCRIPTION
**Pull request by Siddharth-coder13**

### CODE REVIEW

File: git_guide.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+- [Getting started](#getting-started)
+- [First contribution](#first-contribution)
+- [Good first bugs](#good-first-bugs)
+
+## Getting started
+To get started with git,first install git from the given link or if you have already installed it then update to its latest version.
+- [Install git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+
+After installing git successfully, make a account at github.
+- [Get started with GitHub](https://help.github.com/en/github/getting-started-with-github/signing-up-for-a-new-github-account)
```
Comment (mcomella) [inline git_guide.md:13]: This links to help guide that tells the user to "Go to GitHub's Pricing page." and "Read the information about the different products and subscriptions that GitHub offers, then click the upgrade button under the subscription you'd like to choose.": we probably just want to send them to the "create free account" page to keep things simple.

File: git_guide.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Git guide
+This guide is designed to help you get started with git and use its basics commands.
+
+- [Getting started](#getting-started)
+- [First contribution](#first-contribution)
+- [Good first bugs](#good-first-bugs)
+
+## Getting started
+To get started with git,first install git from the given link or if you have already installed it then update to its latest version.
```
Comment (mcomella) [inline git_guide.md:9]:
```suggestion
To get started with git, first install git from the given link or if you have already installed it then update to its latest version.
```

File: git_guide.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+After installing git successfully, make a account at github.
+- [Get started with GitHub](https://help.github.com/en/github/getting-started-with-github/signing-up-for-a-new-github-account)
+
+## First contribution
+Making a contribution is always good way to start in GitHub.
+Let's make your first contribution using some simple steps and learning some commands.
+- [Make your contribution](https://github.com/firstcontributions/first-contributions)
+
+## Good first bugs
+Now, you have learned how to make contribution then try some of our easy buge for you.
```
Comment (mcomella) [inline git_guide.md:21]:
```suggestion
Now, you have learned how to make contribution then try some of our easy bugs for you.
```

File: git_guide.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+- [good first](https://github.com/mozilla-mobile/shared-docs/contribute)
```
Comment (mcomella) [inline git_guide.md:22]:
```suggestion
- [Our tips for finding easy bugs](https://github.com/mozilla-mobile/shared-docs/blob/master/android/CONTRIBUTING_code.md#finding-issues-to-work-on)
- [fenix `good first issue`s](https://github.com/mozilla-mobile/fenix/labels/good%20first%20issue)
```

Comment (mcomella) [reply git_guide.md:22]: Changes I made:
- the `.../contribute` link appears to have more than just `good first issue` so I wanted to specifically link to those
- Fenix is our main project so I changed the repo from `shared-docs` to Fenix
- I also linked to existing documentation we have on finding good first issues

File: git_guide.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+To get started with git,first install git from the given link or if you have already installed it then update to its latest version.
+- [Install git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+
+After installing git successfully, make a account at github.
+- [Get started with GitHub](https://help.github.com/en/github/getting-started-with-github/signing-up-for-a-new-github-account)
+
+## First contribution
+Making a contribution is always good way to start in GitHub.
+Let's make your first contribution using some simple steps and learning some commands.
+- [Make your contribution](https://github.com/firstcontributions/first-contributions)
```
Comment (mcomella) [inline git_guide.md:18]: I appreciate that we're linking out to external resources rather than creating our own.

File: git_guide.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
+# Git guide
```
Comment (mcomella) [inline git_guide.md:1]: Can you link to this from the beginner's guides in `CONTRIBUTING_code`? https://github.com/mozilla-mobile/shared-docs/blob/master/android/CONTRIBUTING_code.md#beginners-guides

Comment (Siddharth-coder13) [reply git_guide.md:1]: provide a link in beginners-guides.

**Comment by mcomella** [comment]
This looks great to me, thanks @Siddharth-coder13 !

**Pull request merged**